src/sparsezoo/utils/onnx/external_data.py

```diff
@@ -98,13 +112,15 @@ def save_onnx(
             all_tensors_to_one_file=True,
             location=external_data_file,
         )
+        split_external_data(model_path, max_file_size=max_external_file_size)
         return True
 
-    if model.ByteSize() > onnx.checker.MAXIMUM_PROTOBUF:
+    if model.ByteSize() > DUMP_EXTERNAL_DATA_THRESHOLD:
         external_data_file = external_data_file or EXTERNAL_ONNX_DATA_NAME
         _LOGGER.warning(
             "The ONNX model is too large to be saved as a single protobuf. "
-            f"Saving with external data: {external_data_file}"
+            "Saving with external data, with file chunks of maximum size "
+            f"{DUMP_EXTERNAL_DATA_THRESHOLD / 1e9} GB"
```

Comment:
wrong threshold - also let's downgrade from warning - probably a bit to scary for an expected, default flow